packages/core/src/payment/strategies/braintree/braintree-paypal-payment-strategy.ts

```diff
@@ -179,7 +179,9 @@ export default class BraintreePaypalPaymentStrategy implements PaymentStrategy {
 
             this._braintreePaymentProcessor.preloadPaypal();
         } catch (error) {
-            this._handleError(error);
+            if (error instanceof Error) {
+                this._handleError(error);
+            }
```

Comment:
```suggestion
            }
            throw error;
```

So the error is not swallowed silently